src/passes/RemoveUnusedBrs.cpp

```diff
@@ -92,9 +94,18 @@ static bool tooCostlyToRunUnconditionally(const PassOptions& passOptions,
     return false;
   }
   // Consider the cost of executing all the code unconditionally.
-  const auto TOO_MUCH = 7;
   auto total = CostAnalyzer(one).cost + CostAnalyzer(two).cost;
-  return total >= TOO_MUCH;
+  return total >= TooCostlyToRunUnconditionally;
```

Comment:
Yeah, I had some similar thoughts here myself. This seems like a subtle point, and in the end I wasn't sure what to do, so I left the old case as it is, and just copied the constant.

I'm not sure this should be half, as even though the previous case adds the costs, the real issue is more of a max than an add. That is, if we executed either A or B before, and now we execute A and B, then we care about the additional time spent - which means we add either A or B. In the worst case, we are adding the maximum of them, so that existing add should be a max, I think. And if it's a max, I think the constant should be the same here.

However, I'm not totally sure about all that without lots of benchmarking. Meanwhile, the safer thing is to use the same constant here, which in the worst case will optimize fewer things (but not too many).

Reply:
(Oh, my last sentence is wrong I think, so ignore that part.)

Reply:
Yeah, fair enough about the constant being the same if you use a max. Anyway I guess the initial value was basically just a guess to begin with, so there is no principled reason to adjust it without benchmarks.